decidim-meetings/app/views/decidim/meetings/admin/meetings/_form.html.erb

```diff
@@ -8,7 +8,7 @@
     </div>
 
     <div class="row column hashtags__container">
-      <%= form.translated :editor, :description, class: "js-hashtags", hashtaggable: true, value: @meeting.present? ? present(@meeting).description(all_locales: true) : @form.description.with_indifferent_access %>
+      <%= form.translated :editor, :description, hashtaggable: true, value: @meeting.present? ? present(@meeting).description(all_locales: true) : @form.description.with_indifferent_access %>
```

Comment:
The `js-hashtags` class here is redundant because it's already added by: https://github.com/decidim/decidim/blob/601ede8f393dedcac1556d6a49b39455d8fdc97b/decidim-core/lib/decidim/form_builder.rb#L168